forward43/scraper.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, which_scraper):
 
     def write_to_file(self, projects, filename):
         ''' Write a list of projects to file in the data directory '''
-        filepath = os.path.join(DATA_DIRECTORY, f'{self.which_scraper}-{filename}.json')
+        filepath = os.path.join(DATA_DIRECTORY, f'forward_scraper_{self.which_scraper}-{filename}.json')
```

Comment:
There's a todo here that we can put in another ticket to ensure that new scrapes don't override old scrapes from the previous run. I'll make it into a ticket

Reply:
The idea was that new scrapes should replace the older ones with respect to project states. I see however this is not true for the start_some_good scraper. I agree, we should pick it up at a later point